src/tds/query.c

```diff
@@ -728,12 +728,48 @@ tds_get_column_declaration(TDSSOCKET * tds, TDSCOLUMN * curcol, char *out)
 	size = tds_fix_column_size(tds, curcol);
 
 	switch (tds_get_conversion_type(curcol->on_server.column_type, curcol->on_server.column_size)) {
-	case XSYBCHAR:
+	case XSYBCHAR: //same as SYBLONGCHAR
+		if (!IS_TDS7_PLUS(tds->conn)) { //SYBLONGCHAR
+			switch (curcol->column_usertype) { // dirty temporary hack
```

Comment:
Please don't use C++ style comments. Although all modern C compilers should support it it's not uncommon that someone use very old limited compilers.

Reply:
OK.

Reply:
Fixed